### PR TITLE
1.11 release post.

### PR DESCRIPTION
**Pull request by anthonyshew**
🚢

Closes TURBO-1735

### CODE REVIEW

**Review by chris-olszewski — APPROVED** [review]
LFG

File: docs/pages/blog/turbo-1-11-0.mdx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+- [**Group logs for improved readability**](#group-logs-for-improved-readability): You can now specify `--log-order=grouped` to organize your logs into distinct sections separated by tasks.
+- [**Updated examples**](#updated-examples): Start with an example for Next.js, Svelte, Remix, Nuxt, and more.
+
+Update today by running `npx @turbo/codemod migrate`.
+
+## Unlocking the future of `turbo` with Rust
+
+As we wrote about in [our blog post about the migration](https://vercel.com/blog/turborepo-migration-go-rust), we're excited to start seeing the benefits of our porting effort for Turborepo users. Rust has allowed us to surface more complexity in _our_ code so we're able to create a better experience in _your_ codebase in several ways:
+
+- We've noticed a slight performance improvement for the majority of use cases, even without doing much performance tuning in our code yet. We look forward to diving deeper into making `turbo` even faster soon.
```
Comment (tknickman) [inline docs/pages/blog/turbo-1-11-0.mdx:39]: Should we add some metrics? Feels pretty vague here. How much, and which use cases? Even naming one or two would be helpful in my opinion.

File: docs/pages/blog/turbo-1-11-0.mdx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+- We've been able to catch possible bugs much sooner, as Rust forces us to think about the differences between platforms and environments much more closely than Go.
+- The Rust ecosystem has many powerful crates that served our needs well, showcasing the ecosystem alignment we were lacking in Go.
+
+If you find any problems that you believe are related to the new Rust codepath, you can opt back to the Go codepath using the `--go-fallback` flag and [report an issue on GitHub](https://github.com/vercel/turbo/issues/new?assignees=&labels=kind%3A+bug%2Cowned-by%3A+turborepo%2Cneeds%3A+triage&projects=&template=0-turborepo-bug-report.yml&title=%5Bturborepo%5D+).
```
Comment (tknickman) [inline docs/pages/blog/turbo-1-11-0.mdx:43]: Non blocking - we could create a label `rust-port` (or something) that gets auto added when using this link (set it here via query param).

File: docs/pages/blog/turbo-1-11-0.mdx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+Enable this behavior using `--log-order=grouped`. [Check out the documentation](https://turbo.build/repo/docs/reference/command-line-reference/run#--log-order) to learn more
+
+## Updated examples
+
+Our repository contains [`an examples directory`](https://github.com/vercel/turbo/tree/main/examples) with starter monorepos for your projects.
+
+To get started with an example, run:
+
+```bash
+# Uses the with-basic example
```
Comment (tknickman) [inline docs/pages/blog/turbo-1-11-0.mdx:80]:
```suggestion
# Next.js
```

File: docs/pages/blog/turbo-1-11-0.mdx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+```
+
+We're putting a renewed effort towards ensuring our examples are kept up-to-date. Additionally, we've re-architected our examples to advocate for best practices in monorepos. For example:
+
+- In-editor go-to-definition across packages
+- Dependencies updated to latest versions
+- Tooling like ESLint, Prettier, jest, and vitest pre-configured
+- Hot-reloading across packages
+
+Check out [the examples directory](https://github.com/vercel/turbo/tree/main/examples) for more.
```
Comment (tknickman) [inline docs/pages/blog/turbo-1-11-0.mdx:101]:
```suggestion
[Choose an example](/repo/docs/getting-started/from-example) to get started, or [browse the code](https://github.com/vercel/turbo/tree/main/examples) directly for more.
```

File: docs/pages/blog/turbo-1-11-0.mdx [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+  "chrisolszewski",
+  "nicholasyang",
+  "alexanderlyon",
+]} />
+
+Turborepo 1.11 completes our migration to Rust and ships several developer experience improvements:
+
+- [**Our new Rust foundation**](#unlocking-the-future-of-turbo-with-rust): We've finished the port from Go to lay the groundwork for better performance, improved stability, and new features.
+- [**Group logs for improved readability**](#group-logs-for-improved-readability): You can now specify `--log-order=grouped` to organize your logs into distinct sections separated by tasks.
+- [**Updated examples**](#updated-examples): Start with an example for Next.js, Svelte, Remix, Nuxt, and more.
```
Comment (tknickman) [inline docs/pages/blog/turbo-1-11-0.mdx:31]: These don't match the OG, are we including LSP?

File: docs/pages/blog/turbo-1-11-0.mdx [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+    src: '/images/blog/turbo-1-11-0/log-order-groups-dark.png',
+    props: {
+      width: 682,
+      height: 746.5,
+      className: "hidden dark:block rounded-xl"
+    }
+  }}
+/>
+
+Enable this behavior using `--log-order=grouped`. [Check out the documentation](https://turbo.build/repo/docs/reference/command-line-reference/run#--log-order) to learn more
```
Comment (tknickman) [inline docs/pages/blog/turbo-1-11-0.mdx:71]: Internal links should be relative:

```suggestion
Enable this behavior using `--log-order=grouped`. [Check out the documentation](/repo/docs/reference/command-line-reference/run#--log-order) to learn more
```

**Review by tknickman — CHANGES_REQUESTED** [review]
Few comments, but looks good!

**Review by tknickman — APPROVED** [review]
:legend:

**Pull request merged**